src/main/cpp/t06_cinema.cpp

```diff
@@ -29,6 +29,37 @@
 using namespace std;
 
 int t06_cinema() {
-    
-    return 0;
+
+    int n,m, k;
+
+    cin >> n;
+    cin >> m;
+
+    int arr[n][n];
+
+    for (int i = 0; i < n; i++) {
+        for (int j = 0; j < m; j++) {
+            cin >> arr[i][j];
+        }
+    }
+
+    cin >> k;
+
+    int c_s;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'c_s' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4615271)